bin/node/runtime/src/lib.rs

```diff
@@ -1619,4 +1619,9 @@ mod tests {
 
 		is_submit_signed_transaction::<Runtime>();
 	}
+
+	#[test]
+	fn call_size() {
+		assert!(core::mem::size_of::<Call>() <= 90);
```

Comment:
maybe add a note that if this fails, it means that some argument to call enum has been added which is too big.

Reply:
I added a message